packages/std-contracts/src/test/MudV2Test.t.sol

```diff
@@ -8,5 +8,8 @@ contract MudV2Test is Test {
 
   function setUp() public virtual {
     worldAddress = vm.parseAddress(vm.readFile(".mudtest"));
+    // allows passing AccessControl when writing tables directly through libraries
+    // (this should be combined with `--sender` flag when calling `forge test`)
+    vm.startPrank(worldAddress);
```

Comment:
this does feel like a bit of a "gotcha" in that anyone calling `startPrank` later will get errors

Is there another way to run tests in some sort of custom "context" in which we'd could specify where the store calls go?

Reply:
yeah that would be 2nd solution from https://github.com/latticexyz/mud/issues/940, which allows tests to explicitly specify worldAddress for tables called directly from the test contract.
I'd like @alvrs's opinion on that one, it's a pretty big low-level change

Reply:
wouldn't this approach also mean that regular system calls via the world have the wrong `_msgSender()`?